### PR TITLE
Fix "WithoutKeepAlive" test to run and pass on Linux, plus provide more accurate name

### PR DESCRIPTION
**Pull request by carlbrown**

### CODE REVIEW

File: Tests/HTTPTests/ServerTests.swift [hunk 1]
```diff
@@ -533,6 +535,7 @@ class ServerTests: XCTestCase {
         ("testSimpleHelloEndToEnd", testSimpleHelloEndToEnd),
         ("testRequestEchoEndToEnd", testRequestEchoEndToEnd),
         ("testRequestKeepAliveEchoEndToEnd", testRequestKeepAliveEchoEndToEnd),
+        ("testMultipleSimultaneousRequestsEchoEndToEnd",testMultipleSimultaneousRequestsEchoEndToEnd),
```
Comment (seabaylea) [inline Tests/HTTPTests/ServerTests.swift:538]: Minor nit - should have a space.

Comment (seabaylea) [reply Tests/HTTPTests/ServerTests.swift:538]: There's also whitespace issues on lines 285->287 which could be fixed up at the same time.

**Comment by seabaylea** [comment]
@carlbrown could you make the minor changes above? Then we can get this in.

**Comment by carlbrown** [comment]
This test no longer exists where it was when this PR was opened. Can't merge.

**Pull request closed without merging**